test/oph/ehoks/hoks/hoks_handler_test.clj

```diff
@@ -869,6 +869,47 @@
                             :data
                             :result)))))))))
 
+(deftest test-hoks-shallow-delete-and-undo
```

Comment:
should: tää on aika monimutkainen asia mitä ollaan tekemässä, joten ehkä olisi hyvä olla testitapaukset ainakin seuraaville:
- HOKS luodaan, päivitetään, kevytpoistetaan ja palautetaan
- HOKS luodaan, kevytpoistetaan, palautetaan, päivitetään, kevytpoistetaan, palautetaan
- HOKS luodaan, PATCHataan osia, kevytpoistetaan ja palautetaan
- HOKS luodaan, kevytpoistetaan, palautetaan, päivitetään osia, kevytpoistetaan, palautetaan

Reply:
Jep, lisään testitapauksia!